concepticondata/concepticon.tsv

```diff
@@ -836,7 +836,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 835	CHAPTER	Speech and language	A part of a book, a literary work etc. with a distinct content, set apart using typography.	Person/Thing	
 836	SCREWDRIVER	Modern world	A hand tool used for driving screws.	Person/Thing	
 837	BLUE	Sense perception	Having the pure color of a clear sky; the primary color between green and violet in the visible spectrum, an effect of light with a wavelength between 450 and 500 nm.	Property	
-838	REMAINS	Spatial relations	The physical structure of a dead animal or person.	Person/Thing	
+838	REMAINS	Spatial relations	What remains after other things have been taken away (e.g. eaten or destroyed).	Person/Thing	
```

Comment:
Should it be 'spatial relations'?

Reply:
REMAIN is in spatial relations, too, and this is basically its nominalization.

Reply:
Ok. :+1: